core-bundle/contao/drivers/DC_Folder.php

```diff
@@ -1175,7 +1175,7 @@ public function move($blnIsAjax=false)
 					throw new ResponseException(new Response('', 201));
 				}
 
-				// Do not purge the html folder (see #2898)
+				// Do not purge the HTML folder (see #2898)
```

Comment:
Not that it matters all that much - but the folder was in fact called `html` (not "`HTML`"). But this comment has lost its relevancy since Contao 4, so we could remove it altogether.

```suggestion
```